maplibre/src/render/resource/tracked_render_pass.rs

```diff
@@ -202,8 +202,8 @@ impl<'a> TrackedRenderPass<'a> {
     /// Push a new debug group over the internal stack. Subsequent render commands and debug
     /// markers are grouped into this new group, until [`pop_debug_group`] is called.
     ///
-    /// ```
-    /// # fn example(mut pass: bevy_render::render_phase::TrackedRenderPass<'static>) {
+    /// ```ignore
```

Comment:
Do you think we can fix these doc-tests? Should I take a look?

Reply:
These fail because they're using private types. I think that unless you want to make them public and expose them to the end user, the only viable solution would be to refactor them out to a separate, internal library. In the current state of the project however it may not worth doing it, as the architecture will change a lot probably. So in general, not much effort should be put into documenting internal things in-depth (with examples) currently, and ignoring some of these is fine.

Reply:
Ah yes, true, I think in another PR I just made them public. I'll take care!

This documentaiton was 0-effort because it was copied from the great bevy engine :)